handsontable/src/plugins/comments/comments.js

```diff
@@ -294,6 +295,24 @@ export class Comments extends BasePlugin {
       runOnlyIf: () => this.#editor.isVisible() && this.#editor.isFocused(),
       group: SHORTCUTS_GROUP,
     });
+
+    pluginContext.addShortcut({
+      keys: [['SHIFT', 'TAB'], ['TAB']],
+      callback: (_, keys) => {
+        this.#editor.setValue(this.#commentValueBeforeSave);
+        this.hide();
+
+        manager.setActiveContextName('grid');
+
+        // Use the grid's default callback.
+        gridContext.getShortcuts(keys).filter(
+          shortcut => shortcut.group === GRID_GROUP,
+        ).forEach((shortcut) => {
+          shortcut.callback();
+        });
+      },
+      group: SHORTCUTS_GROUP,
+    });
```

Comment:
I'd follow the same pattern for key strings as the code above and you can simplify the code using the `forwardToContext` option. So the whole thing might look like this:
```suggestion
    pluginContext.addShortcut({
      keys: [['Shift', 'Tab'], ['Tab']],
      forwardToContext: manager.getContext('grid'),
      callback: () => {
        this.#editor.setValue(this.#commentValueBeforeSave);
        this.hide();
        manager.setActiveContextName('grid');
      },
      group: SHORTCUTS_GROUP,
    });
```

Reply:
I couldn't remember the name of the `forwardToContext` option, couldn't find it. That's exactly what I wanted to do 😆